modin/logging/config.py

```diff
@@ -83,14 +86,17 @@ def bytes_int_to_str(num_bytes, suffix="B"):
         Human-readable string format.
     """
     factor = 1000
+    bytes: float = num_bytes
```

Comment:
`bytes` is a built-in keyword, so we shouldn't use it.

Reply:
This is a good point, I'll make the change